rali/CMakeLists.txt

```diff
@@ -72,24 +72,29 @@ if(${BUILD_RALI})
 			    ../amd_openvx/openvx/include/
 			    ../amd_openvx_extensions/amd_rpp/include/
 			    ../amd_openvx_extensions/amd_media/include/
-			    ./include
-			    )
-	link_directories(${AMDRPP_LIBRARIES_DIR} ${TurboJpeg_LIBRARIES_DIR})
+			    ./include)
+			    #./tf/)
+
+	link_directories(${AMDRPP_LIBRARIES_DIR} ${TurboJpeg_LIBRARIES_DIR}
+					/usr/local/lib/)
 	file(GLOB SOURCES "./source/*.cpp")
+			#"./tf/example.pb.cc"
```

Comment:
@LokeshBonta any reason all this TF code is commented out?

Reply:
@kiritigowda ,
Right now, we are working on some changes suggested by Rajy yesterday for the protobuf issue and we will give TF changes as another PR.

Reply:
@kiritigowda  We will remove the commented part.